### PR TITLE
updates FAQ copy on plans page

### PR DESCRIPTION
**Pull request by rralian**
We weren't accounting for our new ability to install plugins and themes, nor were we consistently using the capitalized plan name as is standard. With the updates, it should look like this.

plans FAQ

## Testing
Run this branch and double-check the content at `/plans/:site`. I'd like a sanity check on the content before I merge, so we don't have to create multiple translations requests if this needs any tweaks.

### CODE REVIEW

File: client/my-sites/plans-features-main/index.jsx [hunk 1]
```diff
@@ -243,14 +241,14 @@ class PlansFeaturesMain extends Component {
...
-						' when you upgrade to the premium or business plan.'
+						' when you upgrade to the Personal, Premium, or Business plan.'
 					) }
 				/>
 
 				<FAQItem
 					question={ translate( 'Do you offer email accounts?' ) }
 					answer={ translate(
-						'Yes. If you register a new domain with our premium or business plans, you can optionally' +
+						'Yes. If you register a new domain with our Personal, Premium, or Business plans, you can optionally' +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/index.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Yes. If you register a new domain with our premium or business plans, you can optionally add G Suite. You can also set up email forwarding for any custom domain registered through WordPress.com. {{a}}Find out more about email{{/a}}.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/plans-features-main/index.jsx [hunk 2]
```diff
@@ -213,11 +213,10 @@ class PlansFeaturesMain extends Component {
 				/>
 
 				<FAQItem
-					question={ translate( 'Can I upload my own plugins?' ) }
+					question={ translate( 'Can I install plugins?' ) }
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/index.jsx]: Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Install Plugins' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plans-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/my-sites/plans-features-main/index.jsx [hunk 3]
```diff
@@ -197,7 +197,7 @@ class PlansFeaturesMain extends Component {
 				<FAQItem
 					question={ translate( 'Do you sell domains?' ) }
 					answer={ translate(
-						'Yes! The personal, premium, and business plans include a free custom domain. That includes new' +
+						'Yes! The Personal, Premium, and Business plans include a free custom domain. That includes new' +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/index.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Yes! The personal, premium, and business plans include a free custom domain. That includes new domains purchased through WordPress.com or your own existing domain that you can map to your WordPress.com site. {{a}}Find out more about domains.{{/a}}' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by benhuberman** [comment]
Thanks for the ping @jhnstn / @rralian. A few notes on the copy:

**Can I install plugins?**
- "community plugins" could be misleading / ambiguous. Would "any WordPress plugin" fit here (and be accurate)? Or "external plugins" if we need to stay a bit more restrictive?
- "All plans..." -- I'd reword this sentence / pare it down some: "All plans already come with a custom set of plugins tailored just for them."

**Can I upload my own theme?**
- I propose some tweaks to the "All plans..." sentence: "All plans give you access to our directory of free and premium themes. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team."

**Do you offer email accounts?**
- Would it make sense to say a tiny bit about G Suite? Even just add something like "Google-powered G Suite"?

File: client/my-sites/plans-features-main/index.jsx (hunk 2 above)
Comment (rralian) [inline client/my-sites/plans-features-main/index.jsx]: 👎

File: client/my-sites/plans-features-main/index.jsx (hunk 3 above)
Comment (rralian) [inline client/my-sites/plans-features-main/index.jsx]: 👎

File: client/lib/plans/constants.js [hunk 4]
```diff
@@ -600,7 +600,7 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_UPLOAD_THEMES ]: {
 		getSlug: () => FEATURE_UPLOAD_THEMES,
-		getTitle: () => i18n.translate( 'Upload themes' ),
+		getTitle: () => i18n.translate( 'Upload Themes' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :ok: This change is safe, we'll keep the existing 23 translations.

File: client/my-sites/plans-features-main/index.jsx [hunk 5]
```diff
@@ -243,15 +241,15 @@ class PlansFeaturesMain extends Component {
...
+						' when you upgrade to the Personal, Premium, or Business plan.'
 					) }
 				/>
 
 				<FAQItem
 					question={ translate( 'Do you offer email accounts?' ) }
 					answer={ translate(
-						'Yes. If you register a new domain with our premium or business plans, you can optionally' +
-						' add G Suite. You can also set up email forwarding for any custom domain' +
+						'Yes. If you register a new domain with our Personal, Premium, or Business plans, you can optionally' +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/index.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Yes. If you register a new domain with our premium or business plans, you can optionally add G Suite. You can also set up email forwarding for any custom domain registered through WordPress.com. {{a}}Find out more about email{{/a}}.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (rralian) [inline client/my-sites/plans-features-main/index.jsx]: 👎

**Comment by rralian** [comment]
Thanks for the feedback @benhuberman. I've made the changes you suggested. I also noticed we weren't capitalizing "Install Themes" consistently like we were for other features, so I addressed that. Can you give it another look?

plans text

**Comment by rralian** [comment]
Sidenote: we should say something about themes and plugins in the text at the top for the business plan, but I think that should be a/b-tested and handled separately.

**Comment by apeatling** [comment]
I'm a little nervous about saying you will not lose content if you downgrade, this is may not be true if you have a Business plan and install a plugin that captures and displays content, then downgrade.

We are not offering GSuite now from what I can tell? So that section is incorrect.

**Comment by rralian** [comment]
> I'm a little nervous about saying you will not lose content if you downgrade, this is may not be true if you have a Business plan and install a plugin that captures and displays content, then downgrade.

What if I just remove that sentence?

> We are not offering GSuite now from what I can tell? So that section is incorrect.

We still sell it from `/domains/manage/email/:site`, it's just no longer in the NUX flow. Until/unless we remove it completely, I'd keep this in here.

**Comment by rralian** [comment]
@apeatling how's this?

faq updated

**Comment by benhuberman** [comment]
@rralian Looking good! 
The only additional tweak I'd suggest, now that I see the new version in place, is to nix "optionally" in the email section:

> Yes. If you register a new domain with our premium or business plans, you can ~optionally~ add Google-powered G Suite.

**Comment by rralian** [comment]
@thanks @benhuberman... updated
faq updated

File: client/my-sites/plans-features-main/index.jsx [hunk 6]
```diff
@@ -243,15 +241,15 @@ class PlansFeaturesMain extends Component {
...
+						' when you upgrade to the Personal, Premium, or Business plan.'
 					) }
 				/>
 
 				<FAQItem
 					question={ translate( 'Do you offer email accounts?' ) }
 					answer={ translate(
-						'Yes. If you register a new domain with our premium or business plans, you can optionally' +
-						' add G Suite. You can also set up email forwarding for any custom domain' +
+						'Yes. If you register a new domain with our Personal, Premium, or Business plans, you can' +
```
Comment (a8ci18n) [inline client/my-sites/plans-features-main/index.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Yes. If you register a new domain with our premium or business plans, you can optionally add G Suite. You can also set up email forwarding for any custom domain registered through WordPress.com. {{a}}Find out more about email{{/a}}.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (rralian) [inline client/my-sites/plans-features-main/index.jsx]: 👎

**Comment by apeatling** [comment]
:+1:
